src/web/View.tsx

```diff
@@ -82,19 +82,17 @@ export class View extends ViewBase<Types.ViewProps, {}> {
         focusManager: PropTypes.object
     };
 
-    private focusManager: FocusManager;
+    private _focusManager: FocusManager;
 
-    private resizeDetectorAnimationFrame: number;
-    private resizeDetectorNodes: { grow?: HTMLElement, shrink?: HTMLElement } = {};
+    private _resizeDetectorAnimationFrame: number;
+    private _resizeDetectorNodes: { grow?: HTMLElement, shrink?: HTMLElement } = {};
 
     constructor(props: Types.ViewProps) {
         super(props);
-        if (this.props.restrictFocusWithin) {
-            this.focusManager = new FocusManager();
-        }
+        this._focusManager = new FocusManager();
```

Comment:
I'm a bit concerned about the perf and memory implications of this. We allocate a LOT of Views, and most of them don't need a FocusManager. Could we allocate this on demand or conditionally, only when restrictFocusWithin is true? If not, could we allocate it only when mounted?